src/main/java/Line.java

```diff
@@ -0,0 +1,26 @@
+public class Line extends Task {
+    protected String by;
+
+    public Line(String taskDescription, String by) {
+        super(taskDescription);
+        this.by = by;
+    }
+
+    public String getBy() {
+
+        return this.by;
+    }
+
+    public void setBy(String by) {
+        this.by = by;
+    }
+
+    @Override
+    public String toString() {
+        return "[D]" + super.toString() + "(by: " + this.by + ")";
+    }
+    @Override
```

Comment:
You could consider using consistent spacings between each methods, this applies to all other files too